source/releasenotes/2025-03-03-terminus-362.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Terminus 3.6.2 release now available"
+published_date: "2025-03-03"
+categories: [tools-apis]
+---
+
+Terminus [3.6.2](https://github.com/pantheon-systems/terminus/releases/tag/3.6.2) is now available as of March 03, 2025. This minor update contains the latest bug fixes and improvements for this tool.
```

Comment:
```suggestion
Terminus [3.6.2](https://github.com/pantheon-systems/terminus/releases/tag/3.6.2) is now available as of March 03, 2025. This small update contains the latest bug fixes and improvements for this tool.
```

Reply:
The readers who know SemVer labelling will be confused because in SemVer `3.6.2` is a patch release, minor would be `3.7.0`. But I don't expect most readers to make that distinction so I'm doing with "small"

Reply:
fwiw I simply cloned the previous release note and updated the versions.  I'll update our terminus release docu to point to this one as a starting point.